source/codegen/format-mi-metadata.py

```diff
@@ -0,0 +1,115 @@
+import argparse
```

Comment:
I can delete this file. It was just temporarily needed to auto-format all the mi-driver metadata files.

It doesn't hurt to keep it around. Still builds fine but might be noisy to keep it around. Thoughts, @astarche ?

Reply:
What about adding it as a workflow action? to run this and fail if it would change any MI drivers?

Anyway, I think I slightly prefer keeping it. We have a number of utility scripts and they're useful when they're useful.

Reply:
I'll go ahead and submit it since it's useful when it's useful and doesn't harm anything like you said.

True, after we update the MI drivers we could put this in a workflow as a verification step. I'll make a note to circle back on that.